docs/docs/getting-started.server.mdx

```diff
@@ -506,7 +506,47 @@ Which in turn lets us choose whether to use the `@mdx-js/mdx` or
 through webpack loader syntax in imports.
 
 Install the webpack loader [`@mdx-js/loader`][mdx-loader].
-There is no need to configure it.
+
+To enable direct MDX imports w/o `!@mdx-js/loader!` prefix, the loader can
+further be added to webpack’s config, using a
+[`react-scripts`](http://github.com/facebook/create-react-app/tree/main/packages/react-scripts)
+rewiring tool, e.g. [CRACO](http://github.com/gsoft-inc/craco).
+
+<details>
+  <summary>Expand CRACO example</summary>
+
+  ```js path="craco.config.js"
+  const { addAfterLoader, loaderByName } = require('@craco/craco')
+
+  module.exports = {
+    webpack: {
+      configure: (webpackConfig) => {
+        addAfterLoader(webpackConfig, loaderByName('babel-loader'), {
+          test: /\.(md|mdx)$/,
+          loader: require.resolve('@mdx-js/loader')
+        })
+
+        return webpackConfig
+      }
+    }
+  }
```

Comment:
```suggestion
  const {addAfterLoader, loaderByName} = require('@craco/craco')

  module.exports = {
    webpack: {
      configure(webpackConfig) {
        addAfterLoader(webpackConfig, loaderByName('babel-loader'), {
          test: /\.mdx?$/,
          loader: require.resolve('@mdx-js/loader')
        })
        return webpackConfig
      }
    }
  }
```

- match project code style
- use normal function, there’s no reason to use an arrow function
- use the same regex used in other places